### PR TITLE
Fix for RDBC-377

### PR DESCRIPTION
**Pull request by myarichuk**
Fix issue with Inflector so it properly pluralizes names like User -> Users

### CODE REVIEW

File: Raven.CppClient/Inflector.cpp [hunk 1]
```diff
@@ -28,8 +28,8 @@ namespace ravendb::client::impl
 
 	Inflector::FillRules::FillRules()
 	{
-		add_plural("(.*)$", "$1s");
-		add_plural("(.*)s$", "$1s");
+        add_plural("$", "s");
```
Comment (ml054) [inline Raven.CppClient/Inflector.cpp:31]: indent is not correct?

Comment (ml054) [reply Raven.CppClient/Inflector.cpp:31]: looks like old code were using tabs instead of spaces

Comment (myarichuk) [reply Raven.CppClient/Inflector.cpp:31]: It is possible. I have comitted this from Linux, so I think the usual hooks to convert tabs to whitespaces weren't there

Comment (ppekrol) [reply Raven.CppClient/Inflector.cpp:31]: add editor config to handle this, we do not not use hooks for few years now

**Pull request closed without merging**